scripts/Autosomal_QC/1.merge_genotype_batches.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#SBATCH --time=05:00:00
+#SBATCH --ntasks=1
+#SBATCH --cpus-per-task=1
+#SBATCH --mem-per-cpu=64G
+
+#############################################
+### Merge multiple batches of GSA genotyping
+### date=12-04-2019
+### version 0.02 (unofficial)
+### author: EL
+#############################################
+### New
+## 26-04-2021
+## commented
+## added line to remove temporary directory
+## changed memory required from 140G to 64G (untested)
+#############################################
+
+### Load packages
+module load plink
```

Comment:
Wouldn't it be better to point to specific tool versions here? Now it will always load the highest version. But that latest version could become incompatible at one point.

Reply:
Added README file, 
Added version to the modules loaded,